vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -220,7 +220,7 @@ function Install-WindowsPatches {
     # then you can get download links by searching for specific KBs at http://www.catalog.update.microsoft.com/home.aspx
 
     # 2020-11 Cumulative Update for Windows Server 2019 for x64-based Systems (KB4586793)
-    $patchUrls = @("http://download.windowsupdate.com/c/msdownload/update/software/secu/2020/11/windows10.0-kb4586793-x64_99d3826119288e28a7d7e0b61c3c112476ddc87a.msu")
```

Comment:
@AbelHu , forgot to remove the unused patch. Still this can be protected by a VHD building CI check. 😢

Reply:
do you mean that this patch has been installed in the base image?

Reply:
No idea, but we hit an issue when installing this patch
```
    azure-arm: 2020-12-08T08:50:24.8912316+00:00: Downloading windows patch from http://download.windowsupdate.com/c/msdownload/update/software/secu/2020/11/windows10.0-kb4586793-x64_99d3826119288e28a7d7e0b61c3c112476ddc87a.msu to C:\Users\packer\AppData\Local\Temp\windows10.0-kb4586793-x64_99d3826119288e28a7d7e0b61c3c112476ddc87a.msu
    azure-arm: 2020-12-08T08:50:27.3914794+00:00: Starting install of windows10.0-kb4586793-x64_99d3826119288e28a7d7e0b61c3c112476ddc87a.msu
    azure-arm: 2020-12-08T08:51:15.1338096+00:00: Error during install of windows10.0-kb4586793-x64_99d3826119288e28a7d7e0b61c3c112476ddc87a.msu. ExitCode: 2359302
```

Reply:
0x240006 | 2359302 | WU_S_ALREADY_INSTALLED | The update to be installed is already installed on the system. | wuerror.h
-- | -- | -- | -- | --